pkg/virt-api/webhooks/validating-webhook/admitters/vmi-preset-admitter_test.go

```diff
@@ -63,7 +62,9 @@ var _ = Describe("Validating VMIPreset Admitter", func() {
 		),
 	)
 	It("reject invalid VirtualMachineInstance spec", func() {
-		vmi := api.NewMinimalVMI("testvmi")
+		vmi := libvmi.New(
+			libvmi.WithName("testvmi"),
+		)
 		vmiPDomain := &v1.DomainSpec{}
 		vmiDomainByte, _ := json.Marshal(vmi.Spec.Domain)
 		Expect(json.Unmarshal(vmiDomainByte, &vmiPDomain)).To(Succeed())
```

Comment:
Out of scope of this PR: this can be dropped using the `vmi.Spec.Domain` directly and `pointer`.
```golang
		vmiPDomain.Devices.Disks = append(vmi.Spec.Domain.Devices.Disks, v1.Disk{
			Name: "testdisk",
			DiskDevice: v1.DiskDevice{
				Disk:  &v1.DiskTarget{},
				CDRom: &v1.CDRomTarget{},
			},
		})
		vmiPreset := &v1.VirtualMachineInstancePreset{
			Spec: v1.VirtualMachineInstancePresetSpec{
				Domain: pointer.P(vmi.Spec.Domain),
			},
		}
```